src/pyfmi/fmi.pyx

```diff
@@ -625,14 +631,15 @@ cdef class ModelBase:
         return self._log_stream is not None and not self._log_stream.closed
 
     def _open_log_file(self):
-        """ Opens the log file if we are not logging into a given stream. """
+        """ Sets parameter to keep log file open when logging is done to log file, if we are not logging into a given stream. """
         if not self._log_is_stream and self._fmu_log_name != NULL:
-            self.file_object = open(self._fmu_log_name,'a')
+            self._keep_log_open = 1
 
     def _close_log_file(self):
         if self.file_object:
             self.file_object.close()
             self.file_object = None
+            self._keep_log_open = 0
```

Comment:
I propse to always set this in this function. I.e. move self._keep_log_open = 0 outside of the if-statement

Reply:
Good point, I've updated the code with the latest commit